ci/tests/test_geneformer/test_geneformer_model.py

```diff
@@ -169,4 +175,12 @@ def test_cls_eos_tokens_presence(self, geneformer, mock_data):
 
     def test_model_input_size(self, geneformer):
         assert geneformer.config["input_size"] == geneformer.configurer.model_map[geneformer.config["model_name"]]['input_size']
+
+    def test_fine_tune_classifier(self, geneformer, mock_data, fine_tune_mock_data):
+        tokenized_dataset = geneformer.process_data(mock_data, gene_names='gene_symbols')
+        tokenized_dataset = tokenized_dataset.add_column('labels', fine_tune_mock_data)
+        fine_tuned_model = GeneformerFineTuningModel(geneformer, fine_tuning_head="classification", output_size=1)
+        fine_tuned_model.train(train_dataset=tokenized_dataset, label='labels')
+        assert fine_tuned_model is not None
+        assert isinstance(fine_tuned_model, GeneformerFineTuningModel)
```

Comment:
This test does not check much and is not very clear what the actual goal is. 
Could you leave a comment or crate a follow-up ticket to improve them (for UCE, scGPT and Geneformer)?